env0/resource_environment.go

```diff
@@ -266,6 +266,11 @@ func resourceEnvironment() *schema.Resource {
 				Description: "set an alias for this environment in favor of running VCS commands using PR comments against it. Additional details: https://docs.env0.com/docs/plan-and-apply-from-pr-comments",
 				Optional:    true,
 			},
+			"vcs_pr_comments_enabled": {
+				Type:        schema.TypeBool,
+				Description: "set to 'true' to enable running VCS PR plan/apply commands using PR comments. This can be set to 'true' (enabled) without setting alias in 'vcs_commands_alias'. Additional details: https://docs.env0.com/docs/plan-and-apply-from-pr-comments#configuration",
+				Optional:    true,
```

Comment:
generally it's good practice to set a default value.
But we need to support a tri-state. Setting a default, will enforce it to be only 2 states.